user/about.md

```diff
@@ -0,0 +1,7 @@
+# About
```

Comment:
This is a duplication of `installation/about.md`. Looking a bit deeper.. If I understand the code, you have now added a new deliverable: the installation guide. I don't see the need for that. Can't we just remove user guide OR developer guide? In Alfred API: _user == developer._

(We could work towards 2 deliverables: a user guide and a contributor guide, but this is more for future tickets.)